src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -0,0 +1,192 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, number } from 'prop-types';
+import styled from 'styled-components';
+import path from 'ramda/src/path';
+import SectionLabel from '@bbc/psammead-section-label';
+import StoryPromo, { Headline, Link } from '@bbc/psammead-story-promo';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { C_LUNAR, C_EBON, C_GHOST } from '@bbc/psammead-styles/colours';
+import Image from '@bbc/psammead-image';
+import { getDoublePica } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import {
+  GEL_SPACING,
+  GEL_SPACING_HLF,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import useToggle from '#hooks/useToggle';
+import CpsOnwardJourney from '../CpsOnwardJourney';
+import Grid from '../../components/Grid';
+import data from './recommendations.json';
+
+const StoryPromoWrapper = styled(StoryPromo)`
+  display: grid;
+  margin: ${GEL_SPACING_HLF} ${GEL_SPACING_DBL};
+  background-color: ${C_GHOST};
+`;
+
+const CustomSectionLabel = styled(SectionLabel)`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)};
+  color: ${C_EBON};
+  margin-top: 0;
+  background-color: ${C_LUNAR};
+`;
+
+const SectionLabelWrapper = styled.div`
+  padding: ${GEL_SPACING_DBL};
+`;
+
+const WsojWrapper = styled.div`
+  background-color: ${C_LUNAR};
+  padding-bottom: ${GEL_SPACING};
+`;
+
+const StyledStoryPromoLi = styled(StoryPromoLi)`
+  /* Override various paddings set by StoryPromoLi */
+  padding: 0 !important;
```

Comment:
Is it possible to not use !important please?

Reply:
please can we remove !important?

Reply:
@greenc05 removed.